### DISCUSSION TITLE
Categories

### DISCUSSION THREAD

**Discussion by DubSolid** [linuxserver/Heimdall#1123] [Ideas]
Should be a possibility to make categories such as; "internal services", "external services", "media", "administrative" and so on.

**Comment by gmoseley** [comment]
I'd like to expand on this even.

I imagine "subscreens" would be very useful for this.

Essentially you would have tiles for each category and you could assign tiles to the main page or category tiles. You can hop into the category tiles and back out to the main page.

This feature has made me consider deploying a few instances of Heimdall just to replicate it without figuring out how to code it.

**Comment by KodeStar** [comment]
Something along these lines was implemented in v2.6.0 https://github.com/linuxserver/Heimdall/discussions/1292
